src/modules/data/sound.c

```diff
@@ -362,3 +381,7 @@ uint32_t lovrSoundCopy(Sound* src, Sound* dst, uint32_t count, uint32_t srcOffse
 
   return frames;
 }
+
+void *lovrSoundGetCallbackMemo(Sound *sound) {
+  return sound->callbackMemo;
+}
```

Comment:
Sorry, this is incredibly minor: Can you configure your editor to add newlines to the end of files?  There is a file in the repository root `.editorconfig` that many editors (vim, emacs, vscode, sublime, etc.) can use to automatically do this when editing any of the lovr files.

Reply:
I'll see what I can do sorry